src/screens/SignUp/index.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import SignUpForm from '../../components/SignUpForm';
+import "./styles.scss";
+
+export default function SignUp() {
+  return (
+    <>
+      <div className="container">
+        <SignUpForm />
+        
+      </div>
```

Comment:
```suggestion
      <div className="container">
        <SignUpForm />
      </div>
```
Please remove this empty space.